src/file.rs

```diff
@@ -1,4 +1,4 @@
-use std::error::Error;
+// use std::error::Error;
```

Comment:
Would you mind deleting this line?

Reply:
Sure thing

Reply:
Seems you've uncommented it rather than delete it?

Reply:
Yep seems I've got you wrong =) gone now